app/models/info_request.rb

```diff
@@ -1475,12 +1475,12 @@ def applicable_censor_rules
 
   def apply_censor_rules_to_text(text)
     applicable_censor_rules.
-      reduce(text) { |text, rule| rule.apply_to_text(text) }
+      reduce(text) { |uncensored_text, rule| rule.apply_to_text(uncensored_text) }
```

Comment:
For single line blocks I would lean towards using a single lowercase character for the variables. It's assumed that single line blocks are typically short and simple, so a longer or more descriptive variable name would not provide much benefit.

EG:

```suggestion
      reduce(text) { |t, rule| rule.apply_to_text(t) }
```

or 

```suggestion
      reduce(text) { |t, r| r.apply_to_text(t) }
```